dist/gcc-compatible/Hacl_Streaming_Poly1305_128.c

```diff
@@ -26,7 +26,7 @@
 #include "Hacl_Streaming_Poly1305_128.h"
 
 Hacl_Streaming_Poly1305_128_poly1305_128_state
-*Hacl_Streaming_Poly1305_128_create_in(uint8_t *k)
+*Hacl_Streaming_Poly1305_128_create_in(uint8_t *k1)
```

Comment:
Why the change into k1?

Reply:
the conflict avoidance mechanism of krml is sometimes conservative

I think a module somewhere defines a top-level constant named k

Reply:
also fixed